packages/jss-plugin-default-unit/src/index.js

```diff
@@ -24,7 +24,7 @@ const units = addCamelCasedVersion(defaultUnits)
  * Recursive deep style passing function
  */
 function iterate(prop: string, value: any, options: Options) {
-  if (!value) return value
+  if (!value && value !== 0) return value
```

Comment:
I would replace it with: `if (value == null)`

Reply:
Sure thing, this has been addressed in https://github.com/cssinjs/jss/pull/1413/commits/3efc90e440742085c34f25f6de4bd4f7f8f63641.